README.md

```diff
@@ -39,6 +40,100 @@ Note that politeiawww does not provide HTML output.  It strictly handles the
 JSON REST RPC commands only.  The GUI for politeiawww can be found at:
 https://github.com/decred/politeiagui
 
+## Development 
+
+#### 1. If you haven't install Go, check it out the [docs fot it](https://golang.org/doc/install)
```

Comment:
I would change this to: Install [Go](https://golang.org/doc/install) and [dep](https://github.com/golang/dep), if you haven't already.